src/factory.js

```diff
@@ -39,6 +40,12 @@ export default function factory(React, empty) {
       })
     }
 
+    const isRedirect = FailureComponent !== undefined
```

Comment:
OMG. This is at least embarrassing :-) Fixing it asap. 

It started as `isNotRedirect` but after that I found out that the complement is easier to read. Well..